composer.json

```diff
@@ -16,19 +16,19 @@
     }
   ],
   "require": {
-    "php": ">=7.2",
+    "php": ">=7.3",
     "ext-dom": "*",
     "ext-mbstring": "*",
     "guzzlehttp/guzzle": "^6.3",
     "guzzlehttp/psr7": "^1.5",
     "psr/http-message": "^1.0",
-    "xparse/element-finder": "^0.4 || ^0.5",
-    "xparse/expression-translator": "0.0.*"
+    "xparse/element-finder": "^0.5.1-alpha",
```

Comment:
We should avoid using a fixed version here.
You can only check it. You can specify 0.6 version here